### ISSUE TITLE
Do you have the plan to support LRN op with Caffe Platform?

### ISSUE THREAD

**Issue by BUG1989** [XiaoMi/mace#40]
`Exception: Mace does not support caffe op type LRN yet`

**Comment by liyinhgqw** [comment]
Yes, as you can check the code in ops, we've already implemented LRU for CPU.
But model converter has not supported it yet.

Do you want to use it on CPU or GPU?

**Comment by BUG1989** [comment]
I want to use it on Mali-T764 GPU,Thanks a lots!

**Comment by BUG1989** [comment]
The model provided by the algorithm engineer is based on GoogleNet_v1.......23333

**Comment by liyinhgqw** [comment]
Okay. Please contact us if you want to contribute this task.
